moveit_msgs/RobotState initial_state
geometry_msgs/PoseStamped[] waypoints
geometry_msgs/Vector3 goal_position_tolerance
geometry_msgs/Vector3 goal_angle_tolerance
geometry_msgs/Vector3 path_position_tolerance
geometry_msgs/Vector3 path_angle_tolerance
geometry_msgs/Quaternion path_orientation_constraint
float64 task_space_step_size
float64 max_cspace_jump
float64 planning_time
string group_name
string target_link
uint8 IGNORE_ENVIRONMENT_COLLISIONS=0
uint8 CHECK_ENVIRONMENT_COLLISIONS=1
uint8 CARTESIAN_IK=2
uint8 PLAN=4
uint8 FOLLOW_WAYPOINTS=8
uint8 FOLLOW_ORIENTATION_CONSTRAINTS=16
uint8 path_type
